### ISSUE TITLE
FDS 5.4 vs. 5.5 Large Pressure Difference w/o Zone for Box in Box w/ Hot Gas Inlet

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#1032]
```
While re-running problems in 5.5, I noticed in some cases a 10x increase in
pressure. Distilled the difference down to a simple setup comparing 5.4 svn
5210 with 5.5 svn 6004

Setup is a closed room in a closed domain. Hot N2 inlet on floor. W/o a
ZONE around the inner room, FDS 5.5 shows much higher pressure than 5.4. W/
a ZONE the results are similar.  Also, FDS 5.5 w/o ZONE takes 3-4x longer
runtime. See attached input and results.
```

Original issue reported on code.google.com by `grgrgrknght` on 2010-04-12 21:09:53

<hr>
- _Attachment: [b5p5.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1058/comment-0/b5p5.fds)_
- _Attachment: fds_5p5_vs_5p4.png<br>![fds_5p5_vs_5p4.png](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1058/comment-0/fds_5p5_vs_5p4.png)_

**Comment by gforney** [comment]
```
I'll take a look at it.
```

Original issue reported on code.google.com by `mcgratta` on 2010-04-13 13:48:38

**Comment by gforney** [comment]
```
This is a feature, not a bug ;)

Based on your previous example (Issue 1049), we improved the treatment of boundary

conditions at solid walls. Now, in FDS 5.5, this is what's happening. First, 
consider the equation of state:

PV=nRT

When you use the ZONE line, FDS asserts a different background pressure (P) in each

compartment, and each compartment has its own volume (V). When you don't use ZONE,

FDS takes the entire domain as the volume (V), but it assumes no mass flux at solid

walls. So we have the erroneous situation where volume is relieved through the wall,

but mass is not. 

With ZONE, inside the small compartment, P goes up, V stays the same, n and T go up.

Without ZONE, V goes down (in a sense). The compartment pressure increases more to

compensate. 

Bottom line is that ZONE must be used. Eventually, we'd like to have FDS figure out

the ZONEs automatically, but when you have complicated structures in multiple 
meshes, this job is very tricky. So for now, always use ZONEs.

I'm going to put this case On Hold to remind myself to figure out a strategy to 
automatically determine ZONEs.
```

Original issue reported on code.google.com by `mcgratta` on 2010-04-13 14:13:06

**Comment by gforney** [comment]
```
Makes sense. Often my closed volume cannot be enclosed by a rectangular
parallelepiped, so the auto ZONE idea appeals. I once wrote a Dijkstra path finding
code that established a connectivity graph between cell centers using cell face
obstructions to break links. The user picks a point in the enclosure, FDS runs
Dijkstra, any cell that has a valid path is in that enclosure. Repeat for additional
user points in other enclosures. Make sure no cell is in two zones. Unfortunately not
totally automatic.
```

Original issue reported on code.google.com by `grgrgrknght` on 2010-04-13 14:25:10

**Comment by gforney** [comment]
```
I have not had the time to try to completely automate the designation of ZONEs. One
convenience is that you do not have to completely enclose the ZONE with coordinates
XB. That means that the user-specified XB will expand to fill the whole ZONE. But FDS
still needs to parse the ZONE lines to determine where the ZONEs are to be. So I will
say that this issue is fixed. Always use ZONE statements to make it clear to FDS where
the pressure is intended to rise.
```

Original issue reported on code.google.com by `mcgratta` on 2012-05-17 19:19:07

**Comment by gforney** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `mcgratta` on 2012-07-21 19:31:55
